profiles/dev/user.clj

```diff
@@ -36,3 +36,11 @@
          [:message-page :cache-time]
          (constantly new-cache-time))
   true)
+
+(comment
+  (go)
+  (reset)
+  (reset-all)
+  (use 'clojurians-log.repl)
+  (load-demo-data! "../clojurians-log-demo-data")
+  #_:end)
```

Comment:
what's this `#_:end` for?

Reply:
Should I remove this admittedly confusing #_:end)? 
In my defence: Moving the closing bracket to the next line makes evaluating of the (load-demo-data ...) much easier, as I just move to eol with $ before before pressing ,ee. The #_:end is just so that the bracket is not alone, sorry for having to bear with my sentimentality :-), should have cleaned this before committing, right?

Reply:
I took it out. It's not problematic as such, but it might confuse people.

Reply:
Thank you for this elegant solution.